### PR TITLE
Several tweaks to interactive sizing

### PR DESCRIPTION
**Pull request by scytacki**
- figure out if the theme has a pinned header, if it does not then pin the interactive to the top of the screen instead of with a margin. Perviously the interactive would always be pinned with a margin for the header even if the theme didn't have one.
- include the size of the pinned header (if there is one) in the maxHeight calculation in full width layout of interactive box and and assessment block interactives. This maxHeight calcuation is used when an interactive is set to "use all available space" or "default aspect ratio, or set by interactive".
- remove special handling of multiple interactives in the interactive box. Currently when there are multiple interactives, the height limiting is disabled. Note: with the handle of multiple interactives, the height limiting is only based on the window height. So if you embed multiple interactives it is unlikely that the interactive box will be pinned.
- match the width of wrapping plugins to the width of the interactive.
- update the interactive size when a wrapping plugin updates the wrapping content

Additionally this removes a random `<br>` that was at the end of the interactive.
This might be related to the fact that interactive iframe has a display:inline instead of display:block
Removing it fixed some extra whitespace and I couldn't see any negatives.

This includes several updates to comments as well.

[#173456421]

### CODE REVIEW

File: app/assets/javascripts/runtime-init.js [hunk 1]
```diff
@@ -1,6 +1,22 @@
 /*jslint browser: true, sloppy: true, todo: true, devel: true, white: true */
 /*global $ */
 
+// Some themes pin the header and some do not
+var _headerIsPinnedInThisTheme = null;
+function headerIsPinnedInThisTheme() {
+  // This currently seems to be the best way to check this. Themes just change
+  // CSS so there isn't a way for them add content to the dom or run javascript
```
Comment (kswenson) [inline app/assets/javascripts/runtime-init.js:8]:
```suggestion
  // CSS so there isn't a way for them to add content to the dom or run javascript
```

File: app/assets/javascripts/runtime-init.js [hunk 2]
```diff
@@ -1,6 +1,22 @@
...
+  // This is fragile though because someone might mess with .conten-hdr and then
```
Comment (kswenson) [inline app/assets/javascripts/runtime-init.js:9]:
```suggestion
  // This is fragile though because someone might mess with .content-hdr and then
```

**Comment by scytacki** [comment]
This page demonstrates how the extra header space is no longer there when pinning the interactive:
https://lara-demo.concordqa.org/activities/20007/pages/304541
You can compare that to this page:
https://authoring.concord.org/activities/9620/pages/124107
The extra space you see on production is to handle the pinned navigation you see here:
https://lara-demo.concordqa.org/activities/20008/pages/304557

**Comment by scytacki** [comment]
I just found out that the waters theme does not seem to be pinned, but the computation I'm using to determine the header size is not figuring this out. You can currently see the issue here:
https://lara-demo.concordqa.org/activities/20009/pages/304573

**Comment by scytacki** [comment]
Ethan is going to change how the Waters theme disables the navigation pinning, so it matches the other themes that disable the navigation pinning.  So that should fix the waters issue.

**Comment by scytacki** [comment]
Piotr reviewed this on slack:
> I looked at it today more carefully and it seems good to me. Ofc it would be nice to have even less edge cases, but I guess we just need all that having pinning, wrapping plugins, and all the sizing options. Things seem nicely described in this comment and changes make sense to me.

**Pull request merged**